pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1333,6 +1332,22 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(len(causes)).To(Equal(1))
 			Expect(causes[0].Field).To(Equal("fake.domain.devices.interfaces[0].name"))
 		})
+		It("should accept a bridge interface on a pod network when the default network interface is bridge", func() {
+			vm := v1.NewMinimalVMI("testvm")
+			vm.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultBridgeNetworkInterface()}
+			vm.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
+			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vm.Spec, config)
+			Expect(len(causes)).To(Equal(0))
+		})
+		It("should reject a bridge interface on a pod network when the default network interface is not bridge", func() {
+			vm := v1.NewMinimalVMI("testvm")
+			vm.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultBridgeNetworkInterface()}
+			vm.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
+			config.SetDefaultNetworkInterface("masquerade")
```

Comment:
Adding this set method just for the test is not nice. I suggest to use `testutils.UpdateFakeClusterConfig` with a ConfigMap containing the new default interface, similar to how this config was created at the top of these tests. Also, please reset it to the old value after running the test, using `defer()`

Reply:
Hi Marc. Thank you for the note. about the defer(): there is an AfterEach() function at the beginning of the file which I think handle the resting. 
https://github.com/kubevirt/kubevirt/blob/6ca3c3856887915157baab97a26c6177108de94b/pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go#L65

Reply:
@slintes

Reply:
oh, you're right, I missed that one :+1:

Reply:
btw, just above that is a enableSlirpInterface func, which shows how to update the config :)

Reply:
ah, you already did that :+1: